client/app/services/on-error.js

```diff
@@ -13,8 +13,6 @@ export default Ember.Service.extend({
 
       if (originalHandler) {
         originalHandler(error);
-      } else {
-        throw error;
```

Comment:
If you're getting an error thrown from this line, that's expected because this code catches all unhandled errors and re-throws them. The real source of the error would be elsewhere. If this is interfering with debugging, try temporarily commenting out line 15 of `app/routes/application.js`, which will disable the custom error handling and let the error be thrown from its true location.